Student/StudentUITests/Submissions/SubmissionComments/SubmissionCommentsTests.swift

```diff
@@ -69,6 +105,47 @@ class SubmissionCommentsTests: StudentTest {
         XCTAssertTrue(SubmissionComments.textCell(commentID: "2").isVisible)
     }
 
+    func testAudioComments() {
+        let testm4a = Bundle(for: SubmissionCommentsTests.self).url(forResource: "test", withExtension: "m4a")!
+        mockData(GetSubmissionRequest(context: course, assignmentID: assignment.id.value, userID: "1"), value: APISubmission.make([
+            "id": "1",
+            "attempt": 1,
+            "user_id": "1",
+            "submission_type": "online_upload",
+            "attachments": [],
+            "submission_comments": [
+                APISubmissionComment.fixture([
+                    "id": "1",
+                    "author_id": "2",
+                    "author": APISubmissionCommentAuthor.fixture([ "display_name": "Teacher" ]),
+                    "media_comment": APIMediaComment.fixture([
+                        "media_type": "audio",
+                        "url": testm4a.absoluteString,
+                    ]),
+                ]),
+                APISubmissionComment.fixture([
+                    "id": "2",
+                    "author_id": "1",
+                    "author": APISubmissionCommentAuthor.fixture([ "display_name": "Student" ]),
+                    "media_comment": APIMediaComment.fixture([
+                        "media_type": "audio",
+                        "url": testm4a.absoluteString,
+                    ]),
+                ]),
+            ],
+        ]))
+        mockDataRequest(URLRequest(url: testm4a), data: try! Data(contentsOf: testm4a))
+
+        show("/courses/\(course.id)/assignments/\(assignment.id)/submissions/1")
+        SubmissionDetailsElement.drawerFilesButton.tap()
```

Comment:
Is there a reason you tap the files button?

Reply:
I don't need to do that, I can tap the gripper instead. It was just a way to open the drawer.

Reply:
oh, it looks like I change that in my text comment commit. I also fix an issue with the comments not rendering until you change tabs.

Reply:
👍